site/content/en/docs/tasks/failure_policy.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Failure Policy"
+linktitle: "Failure Policy"
+weight: 10
+date: 2025-07-23
+description: >
+    Configuring jobset failure policies
+no_list: true
+---
+
+JobSet provides failure policy API to control how your workload behaves in response to child Job failures.
+
+The `failurePolicy` is defined by a set of `rules`. For any job failure, the rules are evaluated in order, and the first matching rule's action is executed. If no rule matches, the default action is `RestartJobSet`, which counts towards the `maxRestarts` limit.
+
+## Failure Policy Actions
+
+### `FailJobSet`
```

Comment:
I like Kueue's approach for this better than ours..

https://kueue.sigs.k8s.io/docs/tasks/run/jobsets/#example-jobset

If you look in their source, they are inline the yaml in the doc. I think this looks cleaner and doesn't require people to click around a bunch of links.

WDYT?

cc @andreyvelich @mimowo @ahg-g

Reply:
Thank you! This looks a lot better!